web/html/src/branding/css/susemanager/components/text.less

```diff
@@ -0,0 +1,7 @@
+// These are all leftovers from the old theme, hopefully we can drop most of this over time
+.text-muted {
```

Comment:
Do we still need this if we're removing the class from the DOM? Same for the scss one.

Reply:
Yes, we still have use cases where we are using this class from scss file. But I will remove this text.less file I think we don't need it.